### DISCUSSION TITLE
Access session from workflow

### DISCUSSION THREAD

**Discussion by DoNck** [argoproj/argo-workflows#5845] [Q&A]
Hi,

From the argo workflows page ".../userinfo", I can see user info from the SSO (subject, groups).
Is it possible to access these variables from a workflow or workflow template ?

Best regards

**Comment by alexec** [comment]
Short answer no, not as variables. Long answer - what do you want to do?

**Reply by DoNck** [reply]
The container I run in Argo produces log messages on its standard output.
For tracing purpose, I want to add the username information to this message.

**Reply by tooptoop4** [reply]
@alexec when a user logs in and runs an existing workflow template I want the wftemplate to access the variable of that logged in user and have the pod commands produce different result depending on who submitted the wftemplate

**Reply by zensolution** [reply]
For audit purpose, I also want to log who is running the workflow template. I see there is a variable workflow.uid. but it is just a random string. I don't know how to connect to a meaningful username.

**Reply by jandyalaphani** [reply]
I am also looking for same usecase are you able to solve the issue ?
